yaas/blueprint.py

```diff
@@ -1,137 +1,54 @@
 # coding: utf-8
 
+""" Interact with Ambari blueprints. """
+
 from __future__ import absolute_import
 from __future__ import print_function
 
-import argparse
-import inspect
 import json
 import pprint
 import requests
 import sys
 
-from . import common
-
-
-def command(parser, args):
-    """ Interact with Ambari blueprints. """
-    subparsers = parser.add_subparsers(dest='subcommand')
-    subcommands = {
-        'add': _add,
-        'list': _list,
-        'remove': _remove,
-        'show': _show,
-        }
-    for name, fn in subcommands.items():
-        subparsers.add_parser(name, help=inspect.getdoc(fn))
-    subargs, extra = parser.parse_known_args(args)
-    subcommands[subargs.subcommand](
-        subparsers.choices[subargs.subcommand],
-        extra)
+from . import config
+from . import utils
 
-
-def _add(parser, args):
+def add(blueprint, blueprint_name, validate_topology=True):
```

Comment:
Well maybe not--I misread. So Server would be something like:

```python
class Server:
  blueprint = yaas.Blueprint()
  host = yaas.Host()
```

Would the sub objects be constructed with the same arguments as yaas.Server? How would they know the config values?